src/frontend/packages/lib_components/src/hooks/stores/index.ts

```diff
@@ -4,3 +4,4 @@ export * from './useCurrentUser';
 export * from './useJwt';
 export * from './useMaintenance';
 export * from './useInfoWidgetModal';
+export * from './useResponsive';
```

Comment:
The `store` folder is more related to the states (zustand), I would have put `useResponsive` one step higher (`./src/hooks/useResponsive`).